README.md

```diff
@@ -8,16 +8,397 @@ so don't try to use it for anything close to a production environment.
 
-## Quick Start
+<!-- vscode-markdown-toc -->
+* 1. [Operator Deployment](#OperatorDeployment)
+	* 1.1. [Deploy the operator on your cluster](#Deploytheoperatoronyourcluster)
+	* 1.2. [Run](#Run)
+		* 1.2.1. [Metadata server](#Metadataserver)
+		* 1.2.2. [Deployment manager server](#Deploymentmanagerserver)
+		* 1.2.3. [Resource server](#Resourceserver)
+* 2. [Local Deployment Start](#LocalDeploymentStart)
+	* 2.1. [Build binary](#Buildbinary)
+	* 2.2. [Run](#Run-1)
+		* 2.2.1. [Metadata server](#Metadataserver-1)
+		* 2.2.2. [Deployment manager server](#Deploymentmanagerserver-1)
+		* 2.2.3. [Resource server](#Resourceserver-1)
+		* 2.2.4. [Infrastructure Inventory Subscription Server (Resource Server)](#InfrastructureInventorySubscriptionServerResourceServer)
+		* 2.2.5. [Alarm server](#Alarmserver)
+		* 2.2.6. [Alarm Subscription server](#AlarmSubscriptionserver)
+		* 2.2.7. [Alarm Notification server](#AlarmNotificationserver)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+
+##  1. <a name='OperatorDeployment'></a>Operator Deployment
+
+The ORAN O2 IMS implementation in OpenShift is managed by the IMS operator. 
+It configures the different components defined in the specification: 
+the deployment managers service, the resource server, alert server, subscriptions to resource and alert.
+
+The IMS operator will create an O-Cloud API that will be available to be queried, for instance from a SMO. It
+also provides a configuration mechanism using a Kubernetes custom resource definition (CRD) that allows the hub cluster administrator to configure the different IMS microservices properly.
+
+
+###  1.1. <a name='Deploytheoperatoronyourcluster'></a>Deploy the operator on your cluster
+
+The IMS operator is installed on an OpenShift cluster where Red Hat Advanced Cluster Management for Kubernetes (RHACM), a.k.a. hub cluster, is installed too.
+Be sure you are connected to your hub cluster and run:
 
-### Build binary
+``` bash
+make deploy
+```
+The operator then is installed in the `oran-o2ims-system` namespace.
+
+```bash
+$ oc get pods -n oran-o2ims-system
+NAME                                             READY   STATUS    RESTARTS   AGE
+oran-o2ims-controller-manager-8668794cdc-27xvf   2/2     Running   0          11s
+```
+
+###  1.2. <a name='Run'></a>Run
+
+The ORAN2IMS custom resource is the way to configure the different components of the IMS operator. 
+Notice that the ORAN2IMS object must be placed in one of the [allowed namespaces](https://www.google.com/url?q=https://github.com/openshift-kni/oran-o2ims/blob/main/internal/cmd/operator/start_controller_manager.go%23L127&sa=D&source=docs&ust=1722264090981796&usg=AOvVaw1mnJq0RKXe-dd1os-EfDBP), such as the oran-o2ims. 
+Let’s configure the Metadata server.
+
+```bash
+$ oc create ns oran-o2ims
+namespace/oran-o2ims created
+```
+
+####  1.2.1. <a name='Metadataserver'></a>Metadata server
```

Comment:
This document has been structured to start the servers one at a time, but in practice that is not typically how it would be done.  The Inventory CR would be added with all of the fields set to `true` in one operation.  There is no need to do them one at a time -- it really only inflates how many documentation lines that need to be managed and the number of steps to be tested.  IMHO, it would be sufficient/best to simply 1) add the Inventory CR, and 2) verify the endpoints